plugin-server/src/utils/db/db.ts

```diff
@@ -678,19 +720,90 @@ export class DB {
             res[`group${index}_properties`] = JSON.stringify(properties)
         }
 
-        for (const gi of notHandledIdentifiers) {
-            void this.updateGroupPropertiesCache(teamId, gi.index, gi.key, {})
-            res[`group${gi.index}_properties`] = JSON.stringify({}) // also adding to event for consistency
+        for (const groupIdentifier of notHandledIdentifiers) {
+            void this.updateGroupPropertiesCache(teamId, groupIdentifier.index, groupIdentifier.key, {})
+            res[`group${groupIdentifier.index}_properties`] = JSON.stringify({}) // also adding to event for consistency
+        }
+        return res
+    }
+
+    public async getGroupsCreatedAtFromDbAndUpdateCache(
+        teamId: number,
+        groupIdentifiers: GroupIdentifier[]
+    ): Promise<Record<string, string>> {
```

Comment:
The type here is a bit hard to make use externally. Could we add typings for both of the strings, even if they both just alias to a string?